src/index.html

```diff
@@ -17,35 +17,35 @@
   </head>
   <body>
     <header class="header">
-      <a class="header__logo" href="">
+      <a class="header__logo" href="1">
```

Comment:
It's a strange href. It won't work. Fix it everywhere 
```suggestion
      <a class="header__logo" href="#">
```

https://user-images.githubusercontent.com/99403198/226103380-3f2eaf25-e0e8-4752-b3b9-beb7851faf09.mov